PolicyServicePkg/Include/PolicyInterface.h

```diff
@@ -75,10 +75,77 @@ EFI_STATUS
   IN CONST EFI_GUID *PolicyGuid
   );
 
+/**
+  Callback for a policy notification event.
+
+  @param[in]  PolicyGuid        The GUID of the policy being notified.
+  @param[in]  EventTypes        The events that occurred for the notification.
+  @param[in]  CallbackHandle    The handle for the callback being invoked.
+**/
+typedef
+VOID
+(EFIAPI *POLICY_HANDLER_CALLBACK)(
+  IN CONST EFI_GUID *PolicyGuid,
+  IN UINT32 EventTypes,
+  IN VOID *CallbackHandle
+  );
+
+#define POLICY_NOTIFY_SET        (BIT0)

```

Comment:
Could some comments be added to describe the differences between the Types? 

The set and remove are intuitive, but the finalized seems confusing to me.